.github/workflows/sonar.yaml

```diff
@@ -5,21 +5,21 @@ name: "Sonar Scanner"
 on:
   push:
     paths:
-      - 'patient-search/**'
-      - '.github/workflows/codeql.yaml'
+      - "apps/modernization-api/**"
```

Comment:
This could either be rewritten as 
1. "apps/**", which would run the build process whenever a change within the apps directory is made
2. OR "apps/modernization-api/**" and "app/modernization-ui/**", to run the build process whenever something within these either of these two folders, but NOT patient-listener were made